### DISCUSSION TITLE
Space between buttons in Labels dialog

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#17] [1.1 General Help and Discussion]
Dear Alex, dear community,

Only a small point here:
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/f95a0443-d7f2-4666-9a85-fba92655ce57)
The filter- and Insert-button are overlapping. Of course, it is then easily possible to change the size of the dialog box manually.
This fixes the problem permanentely.
In case my resolution is important:
![grafik](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/8a8cbd91-19be-40dd-a5f0-422f06dd15ee)
WinEdt version: Build: 20230519  (v. 11.1) - 64-bit
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
While transitioning from older versions of Delphi that always used 96dpi in the form designer to a dpi-aware Delphi 11 (and now Delphi 12) some dialogs were accidentally resized (usually shrunk) by the resource compiler. This is one of them that escaped my attention. The problems seem to be fixed in the latest Delphi 12 but a few more cases like this may be discovered during beta testing. They will be fixed...
